deploy/sre-prometheus/100-configure-alertmanager-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-configure-alertmanager-operator-alerts
+    role: alert-rules
+  name: sre-configure-alertmanager-operator-alerts
```

Comment:
For the same reason as with d-a-o, what about renaming that prometheusRule to `sre-configure-alertmanager-operator-offline-alerts`